src/app/shared/components/header/tools/tools.component.html

```diff
@@ -1,9 +1,11 @@
 <div class="tools-wrapper">
     <div class="tools">
+
+        <pt-create-new *ngIf="!(route==='/all-tests')"></pt-create-new>
         <span class="icon-wrap"><img class="img" [src]="imageGlassPath"></span>
```

Comment:
add `alt` attribute to images, it is necessary

Reply:
fixed.